go.mod

```diff
@@ -98,6 +98,8 @@ require (
 	github.com/bhmj/jsonslice v0.0.0-20200323023432-92c3edaad8e2
 	github.com/blabber/go-freebsd-sysctl v0.0.0-20201130114544-503969f39d8f
 	github.com/cenkalti/backoff v2.2.1+incompatible
+	github.com/cespare/xxhash v1.1.0 // indirect
+	github.com/cespare/xxhash/v2 v2.1.2
```

Comment:
We also have `murmur3` hash as a dependency already. Would it work instead of `xxhash`?

Reply:
xxhash is reported to be faster than murmurhash in some article. That is being said, the path where I use it is maybe not that critical and maybe murmurhash will do the job. I'll test it